api/errors.go

```diff
@@ -1,47 +1,118 @@
 package api
 
 import (
+	"context"
 	"fmt"
 	"net/http"
+	"os"
+	"runtime/debug"
 )
 
-func badRequestError(w http.ResponseWriter, fmtString string, args ...interface{}) *HTTPError {
-	err := httpError(http.StatusBadRequest, fmtString, args...)
-	sendJSON(w, err.Code, err)
-	return err
+var genericInternalServerErrorResponse = []byte(`{"code":500,"msg":"Internal server error"}`)
+
+func badRequestError(fmtString string, args ...interface{}) *HTTPError {
+	return httpError(http.StatusBadRequest, fmtString, args...)
 }
 
-func internalServerError(w http.ResponseWriter, fmtString string, args ...interface{}) *HTTPError {
-	err := httpError(http.StatusInternalServerError, fmtString, args...)
-	sendJSON(w, err.Code, err)
-	return err
+func internalServerError(fmtString string, args ...interface{}) *HTTPError {
```

Comment:
hmmm we could do it for each, but I think that it should be a debug capability. It is that way in ATS right now. I'd rather not leak if possible.

I think just coming up with a value here that will be unique would be nice for searching. We can then look up the req ID if we need.